### DISCUSSION TITLE
Native dialog ::backdrop doesn't inherit :root css variables

### DISCUSSION THREAD

**Discussion by bschlenk** [terrazzoapp/terrazzo#194] [General]
HI! My team is using cobalt to generate css variables. I'm doing some research into native `dialog` elements, and found that a dialog's `::backdrop` pseudo element doesn't inherit any css at all, and so none of cobalt's output of using `:root` is available. Is there currently a way to maybe output css as `:root, ::backdrop { /* css variables */}`, or a way to opt just certain styles into being available to `::backdrop`s?

**Comment by drwpow** [comment]
That’s a great observation. I personally haven’t used `::backdrop` yet. But this would be an easy change. What if plugin-css simply let you control that selector? It could be `:root` by default, but you could change to `:root, ::backdrop` or even `.foo` if you wanted.

I’d rather let people control this, because some people may not want `::backdrop` in their CSS. Would that work for your usecase?
